go/vt/vtgate/vindexes/vschema.go

```diff
@@ -248,7 +262,7 @@ func resolveAutoIncrement(source *vschemapb.SrvVSchema, vschema *VSchema) error
 			}
 			t.AutoIncrement.Sequence = seq
 			for i, cv := range t.ColumnVindexes {
-				if t.AutoIncrement.Column.Equal(cv.Column) {
+				if t.AutoIncrement.Column.Equal(cv.Columns[0]) {
 					t.AutoIncrement.ColumnVindexNum = i
```

Comment:
This same 🆗 , but want also to double check that is the right logic on this context.

Reply:
I looked at the usage of the `ColumnVindexNum` variable, and it's unused. There was a time when this was needed because we had code that treated auto-inc columns that were also vindexes differently. It's not the case any more. The auto-inc values are generated completely orthogonally now, which means that it doesn't care whether it's a vindex column or not.

In other words, let's delete the `ColumnVindexNum` member completely.